tests/benchmarks/multicore/idle_spim_loopback/src/main.c

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(idle_spim_loopback, LOG_LEVEL_INF);
+
+#include <zephyr/kernel.h>
+#include <zephyr/drivers/spi.h>
+#include <zephyr/linker/devicetree_regions.h>
+#include <zephyr/pm/device_runtime.h>
+
+#include <nrfs_backend_ipc_service.h>
+#include <nrfs_gdpwr.h>
+
+#define SPI_MODE (SPI_OP_MODE_MASTER | SPI_WORD_SET(8) | SPI_LINES_SINGLE | SPI_TRANSFER_MSB \
+				| SPI_MODE_CPHA | SPI_MODE_CPOL)
+
+static struct spi_dt_spec spim_spec = SPI_DT_SPEC_GET(DT_NODELABEL(dut_spi_dt), SPI_MODE, 0);
+
+#define MEMORY_SECTION(node)                                                           \
+	COND_CODE_1(DT_NODE_HAS_PROP(node, memory_regions),                                \
+		    (__attribute__((__section__(                                               \
+			    LINKER_DT_NODE_REGION_NAME(DT_PHANDLE(node, memory_regions)))))),      \
+		    ())
+
+static uint8_t spim_buffer[32] MEMORY_SECTION(DT_BUS(DT_NODELABEL(dut_spi_dt)));
+
+#define DATA_FIELD_LEN	(4)
```

Comment:
I see weird behaviour when I change data length to 8 bytes (or more) per single transmission.
Current measurements shows "inactivity" while SPI is extensively exchanging data.
For 16 bytes I sow SPI active for 20 ms every 200ms. Remaining 180ms was like IDLE.